content/influxdb/cloud/organizations/bucket-schema/_index.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Manage bucket schema
+seotitle: Manage bucket schema in InfluxDB
+description: Manage bucket schema in InfluxDB using the influx CLI.
+menu:
+  influxdb_cloud:
+    name: Manage bucket schema
+    parent: Manage organizations
+weight: 105
+influxdb/cloud/tags: [buckets, bucket-schema, schema]
+---
+
+Use *bucket-schema* to restrict a bucket's columns and data types.
+
+*Bucket-schema* are measurement schema that enforce the shape of
+your bucket data.
+By default, buckets receive *schema-type* `implicit` and InfluxDB's default
+measurement schema.
+
+Buckets with schema-type `explicit` allow you to assign your own schema.
+These buckets reject write requests that do not obey their bucket-schema.
+
+### Create a bucket schema
+Use the influx CLI to set the schema-type and one or more schema for your bucket:
+
+1. Create a bucket with the `schema-type` flag set to `explicit`.
+
+   ```sh
+   {{< get-assets-text "bucket-schema/bucket-schema-type.sh" >}}
+   ```
+
+   To view your bucket's schema-type, use the [`influx bucket list` command](/influxdb/cloud/reference/cli/influx/bucket-schema/list).
+
+   With schema-type `explicit` but no bucket-schema, the
+   bucket rejects write attempts.
+
+   For example,
+   ```sh
+   influx write --bucket my_explicit_bucket --format=lp "cpu,host=foo usage_user=0.5"
+   ```
+
+   returns the following error:
+
+   ```sh
+   Error: failed to write data: schema: bucket "my_explicit_bucket" contains
+   no measurement schemas
+   ```
+
+2. Create a measurement schema *columns file* using CSV, JSON, or Newline delimited JSON (NDJSON).
+   Define the name, field type, and data type of each column.
+
+   {{< code-tabs-wrapper >}}
+   {{% code-tabs %}}
+   [columns.csv](#)
+   [columns.json](#)
+   [columns.ndjson](#)
+   {{% /code-tabs %}}
+   {{% code-tab-content %}}
+   ```sh
+   {{< get-assets-text "bucket-schema/bucket-schema-columns.csv" >}}
+   ```
+   {{% /code-tab-content %}}
+   {{% code-tab-content %}}
+   ```json
+   {{< get-assets-text "bucket-schema/bucket-schema-columns.json" >}}
+   ```
+   {{% /code-tab-content %}}
+   {{% code-tab-content %}}
+   ```json
+   {{< get-assets-text "bucket-schema/bucket-schema-columns.ndjson" >}}
+   ```
+   {{% /code-tab-content %}}
+   {{< /code-tabs-wrapper >}}
```

Comment:
To get these to format correctly under the line item and not break the list numbering:

```suggestion
    {{< code-tabs-wrapper >}}
{{% code-tabs %}}
[columns.csv](#)
[columns.json](#)
[columns.ndjson](#)
{{% /code-tabs %}}
{{% code-tab-content %}}
```sh
{{< get-assets-text "bucket-schema/bucket-schema-columns.csv" >}}
```
{{% /code-tab-content %}}
{{% code-tab-content %}}
```json
{{< get-assets-text "bucket-schema/bucket-schema-columns.json" >}}
```
{{% /code-tab-content %}}
{{% code-tab-content %}}
```json
{{< get-assets-text "bucket-schema/bucket-schema-columns.ndjson" >}}
```
{{% /code-tab-content %}}
    {{< /code-tabs-wrapper >}}
```